src/explorer.ts

```diff
@@ -200,6 +200,7 @@ class KubernetesWorkloadFolder extends KubernetesFolder {
             new KubernetesResourceFolder(kuberesources.allKinds.job),
             new KubernetesResourceFolder(kuberesources.allKinds.cronjob),
             new KubernetesResourceFolder(kuberesources.allKinds.pod),
+            new KubernetesCRDFolder(kuberesources.allKinds.crd),
```

Comment:
You should be able to use `KubernetesSelectsPodsFolder` here instead of creating `KubernetesCRDFolder`

Reply:
actually @itowlson informs me that your implementation is fine, so resolving this!